pkg/sqlite/load.go

```diff
@@ -130,9 +141,238 @@ func (s *sqlLoader) addOperatorBundle(tx *sql.Tx, bundle *registry.Bundle) error
 		return err
 	}
 
+	if *s.enableAlpha {
+		err = s.addSubstitutesFor(tx, bundle)
+		if err != nil {
+			return err
+		}
+	}
+
 	return s.addAPIs(tx, bundle)
 }
 
+func (s *sqlLoader) addSubstitutesFor(tx *sql.Tx, bundle *registry.Bundle) error {
+
+	updateBundleReplaces, err := tx.Prepare("update operatorbundle set replaces = ? where replaces = ?")
+	if err != nil {
+		return err
+	}
+	defer updateBundleReplaces.Close()
+
+	updateBundleSkips, err := tx.Prepare("update operatorbundle set skips = ? where name = ?")
+	if err != nil {
+		return err
+	}
+	defer updateBundleSkips.Close()
+
+	updateBundleSubstitutesFor, err := tx.Prepare("update operatorbundle set substitutesfor = ? where name = ?")
+	if err != nil {
+		return err
+	}
+	defer updateBundleSubstitutesFor.Close()
+
+	updateBundleReplacesSkips, err := tx.Prepare("update operatorbundle set replaces = ?, skips = ? where name = ?")
+	if err != nil {
+		return err
+	}
+	defer updateBundleReplacesSkips.Close()
+
+	csvName := bundle.Name
+
+	replaces, err := bundle.Replaces()
+	if err != nil {
+		return err
+	}
+	skips, err := bundle.Skips()
+	if err != nil {
+		return err
+	}
+	version, err := bundle.Version()
+	if err != nil {
+		return err
+	}
+	substitutesFor, err := bundle.SubstitutesFor()
+	if err != nil {
+		return err
+	}
+	if substitutesFor != "" {
```

Comment:
this is a major branch, is codecov lying about it not being covered?

Reply:
yeah that doesn't look right to me - at least from the tests I've written. What do you think?

Reply:
I agree, just very confusing.

maybe codecov comments don't keep up with force pushes?